### PR TITLE
Add Go Free Range

### PR DESCRIPTION
**Pull request by floehopper**

### CODE REVIEW

File: src/_companies/europe/uk/london/gofreerange/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+website: https://gofreerange.com/
+careers_page: https://gofreerange.com/
+
+stack:
+  - "rails"
+
+created_at: 2023-03-20 10:24:17
+---
+
+Founded in 2009, Go Free Range is a worker co-operative with some of the most experienced application developers in the UK. We pair our extensive development experience with the complementary non-development skills necessary to give software projects the best chance of success. As worker-owners we have a vested interest in your project: success for you means success for us.
```
Comment (CalvinWalzel) [inline src/_companies/europe/uk/london/gofreerange/index.md:18]: Could you change the description to be written in the first person? This way, we can keep all the descriptions consistent.

**Review by CalvinWalzel — COMMENTED** [review]
Hey @floehopper, thank you for the contribution! I've added one comment, let me know what you think :)

File: src/_companies/europe/uk/london/gofreerange/index.md (hunk 1 above)
Comment (floehopper) [reply src/_companies/europe/uk/london/gofreerange/index.md]: @CalvinWalzel I read the descriptions for the other companies and realised they are all written in the 3rd person, so I've changed ours to use that - I hope that makes sense!

Comment (CalvinWalzel) [reply src/_companies/europe/uk/london/gofreerange/index.md]: That's what I meant, sorry for the confusion 🤦 
Thanks for the adjustment!

**Comment by floehopper** [comment]
@CalvinWalzel Let me know if you need me to do anything else.

**Review by CalvinWalzel — APPROVED** [review]
Looks good to me :)

**Pull request merged**